css/style.css

```
@@ -15,5 +14,178 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.header {
+  display: flex;
+  justify-content: space-around;
+}
+#logo {
+  width: 100px;
+  height: 50px;
+}
+
+.navbar ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+}
+.navbar ul li {
+  padding: 10px;
```

```suggestion
  padding: 0.75rem;
```
We should use relative units like `rem` where possible instead of `px` so that our webpage changes size properly if the user changes their browser default font size setting. The same applies to all the other `px`s in this file